tests/unit_tests/c_wrappers/res/job_queue/test_ext_job.py

```diff
@@ -13,11 +14,15 @@ def test_load_forward_model_raises_on_missing():
     with pytest.raises(
         ConfigValidationError, match="Could not open job config file 'CONFIG_FILE'"
     ):
-        _ = ExtJob.from_config_file("CONFIG_FILE")
+        _ = ExtJob.from_config_file("CONFIG_FILE", use_new_parser=False)
 
+    with pytest.raises(IOError, match="file(.+?)not found"):
+        _ = ExtJob.from_config_file("CONFIG_FILE", use_new_parser=True)
 
+
+@pytest.mark.parametrize("use_new_parser", [True, False])
```

Comment:
Probably a good idea to connect `use_new_parser` up to the `USE_OLD_PARSER_BY_DEFAULT` like we did for workflow_job.